cmd/set_dma_latency/README.md

```diff
@@ -0,0 +1,40 @@
+# set_dma_latency
+
+## Overview
+
+The `set_dma_latency` can be used to set the value of the
+`/dev/cpu_dma_latency` file. This allows us to control, or disable,
+the use of `P` and `C` states on the processor.
+
+This is useful during metrics evaluation, as entering and leaving
+`C` and `P` states incurs some overheads, and thus can inject noise
+into the system.
+
+## Detail
+
+The program takes a single argument (32bit microsecond value), opens
+the `/dev/cpu_dma_latency` file, and writes the value to that file
+as a 32 bit value. The program will then sleep (the file must be
```

Comment:
The program sleeps, but the file must be kept open for the written value to continue to have effect.

Reply:
I've reworded this - pls check.